spec/support/page_objects/teacher_interface/application/active_application.rb

```diff
@@ -0,0 +1,37 @@
+module PageObjects
+  module TeacherInterface
+    class TaskListItem < SitePrism::Section
+      element :link, "a"
+      element :status, ".govuk-tag"
+    end
+
+    class TaskListSection < SitePrism::Section
+      element :heading, "h2"
+      sections :task_list_items, TaskListItem, ".app-task-list__item"
+    end
+
+    class ActiveApplication < SitePrism::Page
```

Comment:
I notice this is in a folder structure `page_objects/teacher_interface/application/active_application` however the Ruby module structure is `PageObjects::TeacherInterface::ActiveApplication`.

Typically, the folder structure and module structure should match, although folders are written with underscores and models are written with uppercase letters. For the class to match the folder structure it should be `PageObjects::TeacherInterface::Application::ActiveApplication`, however in this case I'd suggest removing the `application` folder since there aren't too many different page.